Update input width in _chips.scss sass component

Git-Harshit opened the pull request:
This is a minor but suitable change which updates the width of input field (for adding new chips) from fixed 120px to fit-content, with same !important weight. Also, the min-width and max-width properties are set to limit its size between 100px and 200px. With this change, the input placeholder (if limited to under 200px) will not get cropped, and it will also have a better fit into the input field, thereby offering it more flexibility to adapt the content.

## Proposed changes
<!-- Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue. -->
Basically, this change fixes the issue that I had encountered recently with the placeholder or text of the input before pressing enter to convert it to a chip. Please have a look at how this small change affects the chips input (Screenshot from Google Chrome):

## Screenshots (if appropriate) or codepen:
<!-- Add supplemental screenshots or code examples. Look for a codepen template in our **[CONTRIBUTING document](https://github.com/Dogfalo/materialize/blob/master/CONTRIBUTING.md)**. -->
![Capture-ChangePropose](https://user-images.githubusercontent.com/57865187/83871438-ac1b1d80-a74d-11ea-97ca-673b94d78ad7.PNG)


## Types of changes
<!-- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->

- [ x ] Bug fix (non-breaking change which fixes an issue).
- [ ] New feature (non-breaking change which adds functionality).
- [ ] Breaking change (fix or feature that would cause existing functionality to change).

## Checklist:
<!-- Go over all the following points, and put an `x` in all the boxes that apply. If you're unsure about any of these, don't hesitate to ask. We're here to help! -->

- [ x ] I have read the **[CONTRIBUTING document](https://github.com/Dogfalo/materialize/blob/master/CONTRIBUTING.md)**.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

Git-Harshit committed:
Update input width in _chips.scss sass component

This is a minor commit which updates the width of input field from fixed 120px to fit-content with same !important weight. Also, the optional min-width and max-width are set to limit its size between 100px and 200px. With this change, the input placeholder (if limited to under 200px) will not get cropped, and it will also have a better fit into the input field.

DanielRuf reviewed (changes requested):
Attention, this will not work on IE11. We should keep the old rule and add the new rule after it. So browsers who support it will use the second width rule.

https://caniuse.com/#feat=mdn-css_properties_width_fit-content

Git-Harshit commented:
Yes @DanielRuf sir! Thank you for mentioning! It did actually slip out of my mind when I had made this pull request. I think even the default 'auto' value for width property might not work in the desired way for IE.
Also, can you please tell if the _!important_ weight is really necessary to be used here, as I don't think it is required for the width changes to take effect? 

Now, I guess it's time to update this PR.

DanielRuf commented:
Regarding the new or old width setting? !important is bad in general.

Git-Harshit commented:
> Regarding the new or old width setting? !important is bad in general.

Actually, it was about the old width setting, as there was *!important* used with `width:120px` CSS. Completely agreed, it isn't a good practice to use !important weight in general. 

Still, as it was used in the previous code, I didn't remove it for now. Please let me know if in case it has to be discarded.

Git-Harshit committed:
Update input width in _chips.scss sass component

This commit updates input width in _chips.scss sass component from fixed 120px value to fit-content for browsers (wherever supported), so that the input width takes up just the required amount of width, and having it between `min-width:100px` and `max-width:200px` constraints ensures that the input field will always be within just the required width, and will be now be lesser likely to crop placeholders.

DanielRuf reviewed (approved):
LGTM
